### PR TITLE
Mediator refactor

### PR DESCRIPTION
**Pull request by ianco**
Create manage script and refactor code into acapy sub-directory
Add optional demo controller; add support for play-with-docker

### CODE REVIEW

File: acapy/configs/mediator-auto-accept.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Connections
+debug-connections: true
+auto-accept-invites: true
+auto-accept-requests: true
+auto-ping-connection: true
+
+# Print an admin invite
+connections-invite: true
+invite-metadata: '{"group": "admin"}'
+invite-label: "Mediator (Admin)"
```
Comment (dbluhm) [inline acapy/configs/mediator-auto-accept.yml:19]: Leftovers from old toolbox usage, I think?

File: acapy/configs/mediator.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+open-mediation: true
+enable-undelivered-queue: true
+
+# Connections
+debug-connections: true
+
+# Print an admin invite
+connections-invite: true
+invite-metadata: '{"group": "admin"}'
+invite-label: "Mediator (Admin)"
```
Comment (dbluhm) [inline acapy/configs/mediator.yml:16]: Same here

**Review by dbluhm — COMMENTED** [review]
Couple of questions

**Comment by ianco** [comment]
> Couple of questions

Cleaned up the configs, thanks for the review

**Review by dbluhm — APPROVED** [review]
Looks good! Thanks for contributing such a significant refactor :slightly_smiling_face:

**Pull request merged**